reference/installation/config.json

```diff
@@ -0,0 +1,54 @@
+{
+    "hideMemberFields": [
+        "TypeMeta"
+    ],
+    "hideTypePatterns": [
+        "ParseError$",
+        "List$",
+        "AmazonCloudIntegration",
```

Comment:
When I thought about this problem earlier this week, I was disappointed that we'd need to maintain a list of all enterprise CRDs here. 

I wonder if instead, we can place a kube-builder annotation on all of our OS and/or enterprise CRDs to indicate which they are, then just filter based on that here.

Reply:
That's what I tried to do on my 1st attempt (a while back) and didn't figure it out. This seemed like the easy route to keep the Enterprise resources out, at least until we can figure out the more involved/complicated route of a kube-builder annotation.